data-pipeline/README.md

```diff
@@ -33,29 +37,63 @@ Follow the directions below to set this project up on your local machine.
     cd variant-nowcast-hub/data-pipeline
     ```
 
-2. Make sure you have a version of Python installed that meets the `requires-python` constraint in [pyproject.toml](pyproject.toml).
+2. Create a Python virtual environment:
+
+    ```bash
+    python -m venv .venv
+    ```
+
+    **Note:** the resulting virtual environment will use whatever Python interpreter was active when you ran the command
 
-    **Note:** if you don't have Python installed, PDM can install it for you: `pdm python install 3.12.2`
-3. Install the project dependencies (this will also create a virtual environment):
+3. Activate the virtual environment:
 
     ```bash
-    pdm install
+    source .venv/bin/activate
     ```
 
-To sync project dependencies after pulling upstream code changes:
+    **Note:** the command above is for Unix-based systems. If you're using Windows, the command is:
+
+    ```bash
+    .venv\Scripts\activate
+    ```
+
+4. Install the project dependencies. The following commands can also be used to update dependencies after pulling upstream code changes:
+
+    ```bash
+    # if you're planning to run the scripts without making code changes
+    pip install -r requirements/dev-requirements.txt && pip install -e .
+
+    # if you're planning to make and submit code changes
+    pip install -r requirements/dev-requirements.txt && pip install -e .
+    ```
+
+### Adding new dependencies
+
+This project uses [`pip-tools`](https://github.com/jazzband/pip-tools) to generate requirements files from `pyproject.toml`.
+
```

Comment:
Noting that [pip-tools docs](https://github.com/jazzband/pip-tools?tab=readme-ov-file#installation) indicate that you should install it after activating your virtual env, moving note about this here from above.

Note, I think github is eating the closing ` ``` ` in my suggestion here, can't figure out how to make it not do that...
```suggestion
This project uses [`pip-tools`](https://github.com/jazzband/pip-tools) to generate requirements files from `pyproject.toml`.
To install `pip-tools`, run the following after activating your virtual environment:

```bash
python -m pip install pip-tools
```
```